main.py

```
@@ -41,9 +43,38 @@
+        except Exception as e:
+            logger.warning("Failed to load keystore: %s", e)
+
     sk = SigningKey.generate()
     pk = sk.verify_key.encode(encoder=HexEncoder).decode()
+    
+    os.makedirs(path, exist_ok=True)
+    try:
+        with open(keystore_path, "w") as f:
+            json.dump({"private_key": sk.encode(encoder=HexEncoder).decode()}, f)
+        logger.info("Created new wallet at %s", keystore_path)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)